userguide/virtualmachines.rst

```diff
@@ -271,6 +271,25 @@ and basic settings for the new virtual machine:
    Creating a Sample Virtual Machine
 
 
+.. _Installing Docker:
+
+Installing Docker
+-----------------
+
+To install Docker on %brand%, it must be installed inside of a virtual
+machine.
+
+Choose an operating system and install it on %brand% by following the
```

Comment:
This needs to be more specific: "a *Linux* distro."  Let's recommend Ubuntu, but say that others will also work.